README.rst

```diff
@@ -18,17 +21,36 @@ To use:
     # YouTube OAuth2 secret files, downloaded from Google Console.
     OAUTH2_CLIENT_SECRET='path/to/oauth-client-secret.json'
 
-    # First day of the conference.
-    YEAR='2018'
-    MONTH='6'
-    DAY='1'
-
     # Get talks list API.
     URL='https://tw.pycon.org/2018/ccip/'
 
+
+    # ===== Following are for generate =====
+
+    # YouTube data v3 API key
+    YOUTUBE_API_KEY='AIzaSyAWro8W3P02k9ouJS-1i7znK3tGW7TejsM'
```

Comment:
The `YOUTUBE_API_KEY` should be hidden for security issue. Probably this the key string could be replaced with "YOUR_YOUTUBE_API_KEY"

Reply:
I've revised all sensitive information on README.rst file.